src/components/Counter/Counter.jsx

```diff
@@ -7,39 +8,61 @@ const colors = {
   red: 'rgb(239, 68, 68)',
 };
 
+const initialCount = { count: 0, color: colors.green };
+
+const countReducer = (state, action) => {
+  switch (action.type) {
+    case 'INCREMENT':
+      return { count: state.count + 1, color: state.color };
+    case 'DECREMENT':
+      return { count: state.count - 1, color: state.color };
+    case 'RESET':
+      return { count: 0, color: state.color };
+    case 'CHANGE_COLOR':
+      return {
+        count: state.count,
+        color: action.payload.color,
+      };
```

Comment:
This is amazing Alex! You were able to handle three different color changes with just one action! Awesome!!